lib/tasks/bundlers/generateLibraryPreload.js

```diff
@@ -106,6 +107,73 @@ function getBundleDefinition(namespace, excludes) {
 	};
 }
 
+function getBundleInfoPreloadDefinition(namespace, excludes, coreVersion) {
+	const sections = [{
+		mode: "preload",
+		filters: [
```

Comment:
IMO, this section also needs the default filters, esp. the excludes. If library.js depends on a module that should be excluded from standard preloads, then we should not include it in this section.

With the old configuration, this was not a issue as `resolve:true` was not used.